### ISSUE TITLE
High Contrast theme doubles/triples icons

### ISSUE THREAD

**Issue by alxcm** [Microsoft/azuredatastudio#2719]
- Azure Data Studio Version: 1.0.0

The High Contrast theme included with Azure Data Studio, selectable by default with `Ctrl+K Ctrl+T`, doubles or triples the Run, Cancel, Disconnect, Change Connection, and Explain icons in the Taskbar, ironically making them harder to read. This can be fixed by removing
```css
.hc-black .split-view-view .action-label {
    background: none;
}
```
from lines 646&ndash;648 of `workbench.main.css`, suggesting the theme is incompatible with that parameter.

**Comment by kburtram** [comment]
@alanrenmsft what do you think of the proposed fix?

**Comment by alanrenmsft** [comment]
The logic behind the mentioned CSS class is to not show any icon in toolbar when HC theme is selected. but I don't see this rule in other parts of the product or VS Code. I think the proposed fix is acceptable. I will go ahead and make the change.

Thanks @alxcm !
